sync-core/src/test/java/com/cloudant/sync/replication/BasicPushStrategyTest.java

```diff
@@ -197,26 +208,34 @@ private void push(int expectedDocs) throws Exception {
 
     @Test
     public void push_changeLimitIsOne_batchCounterShouldBeCorrect() throws Exception {
-        currentSetting = new PushConfiguration(1, PushConfiguration.DEFAULT_MAX_BATCH_COUNTER_PER_RUN,
-                PushConfiguration.DEFAULT_BULK_INSERT_SIZE, PushConfiguration.DEFAULT_PUSH_ATTACHMENTS_INLINE);
-        twoDocsCreatedAndThenPushed();
+
+        twoDocsCreatedAndThenPushed(1, BasicPushStrategy.DEFAULT_MAX_BATCH_COUNTER_PER_RUN,
+                BasicPushStrategy.DEFAULT_BULK_INSERT_SIZE, BasicPushStrategy.DEFAULT_PUSH_ATTACHMENTS_INLINE);
         assertPushReplicationStatus(2, 3, "2");
     }
 
-    private void twoDocsCreatedAndThenPushed() throws Exception {
+    private void twoDocsCreatedAndThenPushed( int changeLimitPerBatch,
```

Comment:
You can take remove this variant of twoDocsCreatedAndThenPushed and rely on push() supplying defaults as detailed in my comment above